### PR TITLE
feat: add animated chart bar and column icons

### PR DESCRIPTION
**Pull request by gokh4nozturk**
## I have read the [CONTRIBUTING.md](https://github.com/pqoqubbw/icons/blob/main/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

feature

## Demo

https://github.com/user-attachments/assets/1caebb58-7537-4742-a081-7d569252e550

### CODE REVIEW

**Comment by pqoqubbw** [comment]
lines disappear in the wrong order i guess. could you make them disappear in the right order? from left to right for horizontal chars and from top to bottom for vertical charts

File: icons/chart-bar-decreasing.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+'use client';
+
+import { type Variants, motion, useAnimation } from 'framer-motion';
+import React from 'react';
```
Comment (pqoqubbw) [inline icons/chart-bar-decreasing.tsx:4]: we don't need this import here

File: icons/chart-bar-decreasing.tsx
Comment (pqoqubbw) [inline icons/chart-bar-decreasing.tsx:1]: why do you need `useCallback` in this component?

**Review by pqoqubbw — COMMENTED** [review]
same comments for other files

**Comment by gokh4nozturk** [comment]
Ok, I am on it @pqoqubbw

**Comment by gokh4nozturk** [comment]
hey, @pqoqubbw comments done. I guess.
I'm not sure if the keywords are enough.

**Pull request merged**